server/config/incident.js

```diff
@@ -1,6 +1,25 @@
 module.exports = {
   newIncident: {
-    fields: [],
+    fields: [
+      {
+        locationId: {
+          responseType: 'requiredNumber',
+          validationMessage: 'Where did the incident occur?',
+        },
+      },
+      {
+        forceType: {
+          responseType: 'requiredString',
+          validationMessage: 'Where did the incident occur?',
```

Comment:
same message for both?

Reply:
fixed - we haven't enabled validation yet (it's to be covered in a later story and we are still awaiting real content)